lua/lv-telescope/init.lua

```diff
@@ -36,7 +36,7 @@ telescope.setup {
     file_sorter = require("telescope.sorters").get_fzy_sorter,
     file_ignore_patterns = {},
     generic_sorter = require("telescope.sorters").get_generic_fuzzy_sorter,
-    shorten_path = true,
+    path_display = {"shorten_path"},
```

Comment:
shouldn't this be `shorten` instead of `shorten_path` ? as described [here](https://github.com/nvim-telescope/telescope.nvim/wiki/Path-Display-Configuration#settings)

Reply:
The original issue let me to running the command: :help telescope.changelog-839.
There, they say in the examples:
"...
You can also still pass this to a single builtin call:
```lua
require("telescope.builtin").find_files{
path_display = {"shorten_path}
}
```
"

But I see now that they also mention above the examples:
"
The new way to configure to configure path displays is with:
    `path_display`: It is a table and accepts multiple values:
        - "hidden"    hide file names
        - "tail"      only display the file name, and not the path
        - "absolute"  display absolute paths
        - "shorten"   only display the first character of each directory in
                      the path
"
@abzcoding  I'll rewrite it to shorten.